lib/luaT/luaT.c

```diff
@@ -471,16 +471,15 @@ void luaT_registeratname(lua_State *L, const struct luaL_Reg *methods, const cha
 }
 
 
-/* utility functions */
-const char *luaT_classrootname(const char *tname)
+const char* luaT_classrootname(const char *tname)
```

Comment:
Good point. Currently, it's a non-issue in that `luaT_classmodulename` seems to be single use; however, I agree that clarification here would be desirable. An obvious patch would be to make seperate methods a la `luaT_outerparentname` and `luaT_innerparentname`. Note that the latter method is (in effect) what `traverse_table` accomplished. Please let me know how you would like me to proceed.